src/sentry/sentry_apps/api/parsers/sentry_app.py

```diff
@@ -73,7 +73,10 @@ class SentryAppParser(Serializer):
     status = serializers.CharField(required=False, allow_null=True)
     events = EventListField(required=False, allow_null=True)
     features = serializers.MultipleChoiceField(
```

Comment:
will we deprecate this eventually?

Reply:
Yep, once the UI is migrated and working correctly, I'll drop this field